awx/api/views/__init__.py

```diff
@@ -400,6 +401,34 @@ def get_queryset(self):
         qs = qs.filter(execution_node=po.hostname)
         return qs
 
+class InstanceInstanceLinksList(ParentMixin, ListAPIView):
+
+    name = _("Instance's Instance Links")
+    parent_model = models.Instance
+    model = models.InstanceLink
+    serializer_class = serializers.InstanceLinkSerializer
+    parent_access = 'read'
+
+    def get_queryset(self):
+        po = self.get_parent_object()
+        qs = get_user_queryset(self.request.user, models.InstanceLink)
+        source = qs.filter(source=po.id)
+        target = qs.filter(target=po.id)
+        return source.union(target)
+
+class InstancePeersList(SubListAPIView):
+
+    name = _("Instance Peers")
+    parent_model = models.Instance
+    model = models.Instance
+    serializer_class = serializers.InstanceSerializer
+    parent_access = 'read'
+
+    def get_queryset(self):
```

Comment:
I don't think this override is necessary, what happens if you were to just set `relationship = 'peers'`?